typescript/sdk/src/router/schemas.ts

```diff
@@ -32,6 +32,13 @@ export const RouterConfigSchema = MailboxClientConfigSchema.merge(
   }),
 );
 
+const DestinationGasDomain = z.string();
+const DestinationGasAmount = z.string(); // This must be a string type to match Ether's type
+export const DestinationGasSchema = z.record(
+  DestinationGasDomain,
+  DestinationGasAmount,
+);
 export const GasRouterConfigSchema = RouterConfigSchema.extend({
   gas: z.number().optional(),
+  destinationGas: DestinationGasSchema.optional(),
```

Comment:
What would be the difference between `gas` and `destinationGas`? Would it be just a "cached" way of storing the remote gas configuration on the current chain?

Reply:
`gas` is used to describe a chain's own `gas` value, but is set on the other chains. destinationChain is a derived mapping that lists the _other_ `gas` values.
```
{
  arbitrum: {
    gas: 10
  },
  base: {
    gas: 20
 }
}
```
will result in:
```
{
  arbitrum: {
    destinationGas: {
      [base]: 20
    }
  },
    base: {
    destinationGas: {
      [arbitrum]: 10
    }
  }
}
```